python_ta/typecheck/base.py

```diff
@@ -831,6 +831,8 @@ def _node_to_type(node, locals=None):
         return [_node_to_type(t) for t in node.elts if not isinstance(t, astroid.Ellipsis)]
     elif isinstance(node, astroid.Const) and node.value is None:
         return None
+    elif isinstance(node, astroid.Const):
```

Comment:
You should only do this if the Const node stores a string. Other `Const` values should proceed to the `else`